tests/sysprep_test.go

```diff
@@ -318,8 +318,9 @@ var _ = Describe("[Serial][Sysprep][sig-compute]Syspreped VirtualMachineInstance
 		answerFileWithKey := insertProductKeyToAnswerFileTemplate(answerFileTemplate)
 		windowsVMI = libvmi.New()
 		windowsVMI.Spec = getWindowsSysprepVMISpec()
+		windowsVMI.ObjectMeta.Namespace = testsuite.GetTestNamespace(windowsVMI)
 		cm := libconfigmap.New("sysprepautounattend", map[string]string{"Autounattend.xml": answerFileWithKey, "Unattend.xml": answerFileWithKey})
-		cm, err := virtClient.CoreV1().ConfigMaps(windowsVMI.Namespace).Create(context.Background(), cm, metav1.CreateOptions{})
+		cm, err := virtClient.CoreV1().ConfigMaps(windowsVMI.ObjectMeta.Namespace).Create(context.Background(), cm, metav1.CreateOptions{})
```

Comment:
Isn't `testsuite.GetTestNamespace(windowsVMI)` enough?

Reply:
Yes there was no need for the update to the configmap - i've removed that and tested it here - https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/logs/periodic-kubevirt-e2e-sysprep/1780860358886428672